ISISComputingGroup/EPICS-ioc
# Ticket1631 hifi magnets

ISISComputingGroup/IBEX/issues/1631

### Description of work

Created a lvDCOM IOC for the HIFI Magnets VI.

### To test

The VI is on NDXDEMO so you should be able to connect to that to test the IOC (see the st.cmd).

Note: the VI on NDXDEMO is broken (missing dependencies) but that does not matter for testing the IOC.

### Acceptance criteria

Can read and write to the VI via the IOC

---

#### Code Review


### Functional Tests


### Final steps
